### PR TITLE
chore: address path returned from deriveAddress

### PR DESCRIPTION
**Pull request by slowbackspace**

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -19,6 +19,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
```
Comment (1000101) [inline CHANGELOG.MD:19]: Nothing else has been fixed? :thinking:

Comment (slowbackspace) [reply CHANGELOG.MD:19]: that's exactly right!

File: src/utils/helpers.ts [hunk 2]
```diff
@@ -3,31 +3,55 @@ import {
   BaseAddress,
   NetworkInfo,
   StakeCredential,
+  RewardAddress,
 } from '@emurgo/cardano-serialization-lib-nodejs';
 import EmurgoCip from '@emurgo/cip14-js';
 import { ParseAssetResult } from '../types/utils';
 
+/**
+ * Derives an address with derivation path m/1852'/1815'/account'/type/addressIndex
```
Comment (1000101) [inline src/utils/helpers.ts:12]:
```suggestion
 * Derives an address with derivation path m/1852'/1815'/account'/role/addressIndex
```

File: jest.config.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ module.exports = {
   },
   coverageThreshold: {
     global: {
-      lines: 90,
+      lines: 89,
```
Comment (vladimirvolek) [inline jest.config.js:18]:
```suggestion
      lines: 90,
```

**Review by vladimirvolek — CHANGES_REQUESTED** [review]

**Review by vladimirvolek — APPROVED** [review]

**Pull request merged**